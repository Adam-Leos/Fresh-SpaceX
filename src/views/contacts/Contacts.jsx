// @flow

import React from 'react';
import Copyright from './../../components/copyright/Copyright.jsx';

import './contacts.scss';

function Contacts() {
  return (
    <div>
      <main className="contacts">
        <a className="contacts__link" href="https://www.linkedin.com/in/adam-leos/" rel="noopener noreferrer" target="_blank">LinkedIn</a>
        <a className="contacts__link contacts__link--facebook" href="https://www.facebook.com/4leos" rel="noopener noreferrer" target="_blank">Facebook</a>
        <a className="contacts__link contacts__link--gmail" href="mailto:leos2397@gmail.com">GMail - leos2397@gmail.com</a>
      </main>
      <footer className="footer">
        <Copyright />
      </footer>
    </div>
  );
}

export default Contacts;
