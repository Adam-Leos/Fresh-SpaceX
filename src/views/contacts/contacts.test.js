import React from 'react';
import Contacts from './Contacts.jsx';

test('Contacts is matching snapshot', () => {
  expect(shallow(<Contacts />)).toMatchSnapshot();
});
