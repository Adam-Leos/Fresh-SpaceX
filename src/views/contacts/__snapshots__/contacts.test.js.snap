// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacts is matching snapshot 1`] = `
<div>
  <main
    className="contacts"
  >
    <a
      className="contacts__link"
      href="https://www.linkedin.com/in/adam-leos/"
      rel="noopener noreferrer"
      target="_blank"
    >
      LinkedIn
    </a>
    <a
      className="contacts__link contacts__link--facebook"
      href="https://www.facebook.com/4leos"
      rel="noopener noreferrer"
      target="_blank"
    >
      Facebook
    </a>
    <a
      className="contacts__link contacts__link--gmail"
      href="mailto:leos2397@gmail.com"
    >
      GMail - leos2397@gmail.com
    </a>
  </main>
  <footer
    className="footer"
  >
    <Copyright />
  </footer>
</div>
`;
